lib/mina/puma/tasks.rake

```diff
@@ -23,7 +26,7 @@ namespace :puma do
         if [ -e '#{puma_config}' ]; then
           cd #{deploy_to}/#{current_path} && #{puma_cmd} -q -d -e #{puma_env} -C #{puma_config}
         else
-          cd #{deploy_to}/#{current_path} && #{puma_cmd} -q -d -e #{puma_env} -b 'unix://#{puma_socket}' -S #{puma_state} --pidfile #{puma_pid} --control 'unix://#{pumactl_socket}'
+          cd #{deploy_to}/#{current_path} && #{puma_cmd} -q -d -e #{puma_env} -b 'unix://#{puma_socket}' #{puma_port_option} -S #{puma_state} --pidfile #{puma_pid} --control 'unix://#{pumactl_socket}'
```

Comment:
I think it makes sense to write some kind of syntactic sugar to handle these as the command is getting unwieldy, not to mention sometimes you might want to pass configuration file but still be able to overwrite things with custom settings. Do you think you can take care of that before going further with this pull request?